spec/bundler/shared_helpers_spec.rb

```diff
@@ -261,15 +261,6 @@
       subject.set_bundle_environment
     end
 
-    it "ignores if bundler_ruby_lib is same as rubylibdir" do
```

Comment:
why is this spec removed?

Reply:
This is part of an experiment of the hypothesis that if we never use the $LOAD_PATH, we can revert several patches and workarounds for issues introduced in the past, and we no longer need to special case bundler when manipulating the $LOAD_PATH.

But it's not meant to stay in this PR.